docs/resources/frontend_o11y_app.md

```diff
@@ -42,7 +45,7 @@ resource "grafana_frontend_o11y_app" "test-app" {
 - `allowed_origins` (List of String) A list of allowed origins for CORS.
 - `extra_log_attributes` (Map of String) The extra attributes to append in each signal.
 - `name` (String) The name of Frontend Observability App. Part of the Terraform Resource ID.
-- `settings` (Map of String) The key-value settings of the Frontend Observability app. Available Settings: `{combineLabData=(0|1)}`
+- `settings` (Map of String) The key-value settings of the Frontend Observability app. Available Settings: `{combineLabData=(0|1),geolocation.level=(0|1),geolocation.level=0-4,geolocation.country_denylist=<comma-separated-list-of-country-codes>}`
```

Comment:
Are we able to point to docs which outline what the levels mean?

Reply:
https://grafana.com/docs/grafana-cloud/monitor-applications/frontend-observability/geolocation/#enable-geolocation